packages/lib/event-types/getEventTypesByViewer.ts

```diff
@@ -203,12 +204,24 @@ export const getEventTypesByViewer = async (user: User, filters?: Filters, forRo
     membershipRole: membership.role,
   }));
 
-  const filterTeamsEventTypesBasedOnInput = async (eventType: Awaited<ReturnType<typeof mapEventType>>) => {
+  const filterByTeamIds = async (eventType: Awaited<ReturnType<typeof mapEventType>>) => {
     if (!filters || !hasFilter(filters)) {
       return true;
     }
     return filters?.teamIds?.includes(eventType?.teamId || 0) ?? false;
   };
+  const filterBySchedulingType = (evType: Awaited<ReturnType<typeof mapEventType>>) => {
+    if (!filters || !hasFilter(filters)) {
+      return true;
+    }
+
+    if (!filters.schedulingType) return true;
+
+    if (!evType.schedulingType) return false;
```

Comment:
Question: What is the logic for returning false if event type has no schedulingType ?

If we remove `if (!evType.schedulingType) return false;` from code, then code below:
```
return filters.schedulingType.includes(evType.schedulingType);
```

would perform the same logic, right? So I think that `if (!evType.schedulingType) return false;` can be removed.

Reply:
Nvm fixed this